core/templates/pages/header_css_libs.html

```diff
@@ -2,7 +2,7 @@
 <link rel="preload" href="/third_party/generated/webfonts/fa-brands-400.woff2" as="font" type="font/woff2" crossorigin>
 
 <link rel="stylesheet" type="text/css" media="screen"
-      href="https://fonts.googleapis.com/css?family=Capriola|Roboto|Material+Icons&display=swap">
+      href="https://fonts.googleapis.com/css?family=Capriola|Roboto|Noto+Serif:ital@0;1|Material+Icons&display=swap">
```

Comment:
Why adding new font?

Reply:
As @mschanteltc mentioned in the comment [here](https://github.com/oppia/oppia/pull/9978#issuecomment-663185793), we need to use a serif font which resembles the fonts that get rendered in the editor.